Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

Solaris specific man page changes.

--- top-3.8beta1/top.1.in.orig	2017-07-26 07:11:44.828404508 +0000
+++ top-3.8beta1/top.1.in	2017-07-26 07:13:33.508511863 +0000
@@ -456,8 +456,8 @@
 kill(1),
 ps(1),
 stty(1),
-mem(4),
-renice(8)
+mem(4d),
+renice(1)
 @MAN_SUPPLEMENT@
 .SH COPYRIGHT
 Copyright (C) 1984-2007 William LeFebvre. For additional licensing
--- top-3.8beta1/machine/m_sunos5.man.orig	2017-07-26 07:16:36.542575292 +0000
+++ top-3.8beta1/machine/m_sunos5.man	2017-07-26 07:16:46.741678269 +0000
@@ -51,7 +51,7 @@
 Unlike previous versions of
 .IR top ,
 the swap figures will differ from the summary output of
-.IR swap (1M)
+.IR swap (8)
 since the latter includes physical memory as well.
 .PP
 The column 
